frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -116,7 +123,18 @@ const PVCTableRow = ({ obj, index, key, style }) => {
       <TableData className={tableColumnClasses[4]}>
         {_.get(obj, 'status.capacity.storage', '-')}
       </TableData>
-      <TableData className={tableColumnClasses[5]}>
+      <TableData className={classNames(tableColumnClasses[5])}>
+        {obj?.spec?.storageClassName ? (
```

Comment:
Optional chaining isn't necessary here. `obj.spec` will always be defined

Reply:
@spadgett This PR resolved optional chaining - https://github.com/openshift/console/pull/5382